validation/doc.go

```diff
@@ -0,0 +1,20 @@
+// Package validation contains logic to confirm that an OP Chain is either a valid Standard Chain
+// or a valid Frontier Chain. These checks are provided as Go tests.
+
+// It performs the following checks:
+
+// For Standard Chains:
+
+// * The Chain ID is unique.
+// * There is a valid, public RPC endpoint which returns a valid response.
+// * The resource metering config matches the Standard config.
+//   This includes the resource limit, elasticity multiplier, base fee maximum change denominator, minimum base fee, maximum base fee, and maximum gas.
+// * There is a valid gas price oracle, and the parameters (blob base fee scalar and base feed scalar) are within the permitted range.
+// * The L2 output oracle parameters (submission interval, block time and finalization period) all match the Standard config.
```

Comment:
I think we aren't checking against any Standard config for this one. We actually check against the values used on the OP chain in each superchain https://github.com/ethereum-optimism/superchain-registry/blob/f1be06177cd430401e12992ff7db1ec3608581b7/validation/l2oo_test.go#L61-L66

Reply:
Ah, good catch, I'll update this (in case we decide to merge it)